shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/Address.hs

```diff
@@ -32,6 +32,9 @@ module Shelley.Spec.Ledger.Address
     -- internals exported for testing
     getAddr,
     getKeyHash,
+    BootstrapKeyHash,
```

Comment:
It seems better to define these in `Address.Bootstrap`, but that seemed to create a circular dependency. I think `bootstrapKeyHash` was the root of it, IIRC.